7.1 Snapshot.md

```diff
@@ -0,0 +1,52 @@
+# 7.1 Snapshot
+
+## 7.1.1 Introduction
+This section defines the *Snapshot* protocol.
+
+
+## 7.1.2 Dependencies
+The *Snapshotting* protocol depends on:
+
++ Timestamps: Core module to enable pruning.
++ Tip Selection Algorithm: No messages can be referenced for the part of the Tangle that is being pruned.
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
++ Markers: We require that the part of the Tangle being pruned is not actively dependent on Markers any more. 
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: The past cone check must be relaxed (???TODO)
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise data that is sufficiently far in the past can be removed. We call this process pruning. Thus, the IOTA protocol for a standard node does not attempt to keep all information back until the genesis. Rather the protocol effectively becomes a filter mechanism to proof whether a certain message was at some point in the past included in the ledger. This can be done e.g. through a Merkel tree proof. 
```

Comment:
```suggestion
In order to keep the ledger concise data that is sufficiently old can be removed. We call this process pruning. Thus, the IOTA 2.0 protocol for a standard node does not attempt to keep all information back until the Genesis. Rather the protocol effectively becomes a mechanism to prove whether a certain message was, at some point in the past, included in the ledger. This can be done e.g. through a Merkle tree proof. 
```